tests/store/artifact/test_unity_catalog_models_artifact_repo.py

```diff
@@ -269,3 +269,21 @@ def test_uc_models_artifact_repo_list_artifacts_uses_temporary_creds():
             method="POST",
             json={"name": "MyModel", "version": "12", "operation": "MODEL_VERSION_OPERATION_READ"},
         )
+
+
+def test_get_feature_dependencies():
```

Comment:
this test is just purely to validate that `mlflow.store._unity_catalog.registry.rest_store.get_feature_dependencies` can be called. verified that it fails on current master:

<img width="645" alt="Screenshot 2024-02-06 at 3 19 04 PM" src="https://github.com/mlflow/mlflow/assets/148037680/d6980da0-c23c-457c-8e83-4440890fc625">